openapi/integrator/wallet/server.yaml

```diff
@@ -91,6 +91,19 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /payment-tokens/authorization-response:
+    get:
+      summary: The redirection endpoint the authorization server directs to.
```

Comment:
This does not necessarily need to be used only for _HTTP redirect binding_ only, so we could rephrase to something like this:

> Endpoint used by the authentication provider to provide the authentication response.